app/index.js

```diff
@@ -83,31 +83,31 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'name',
         message: 'What\'s your name:',
         default: this.options.name || this.gitc.user.name,
-        when: this.options.name == null
+        when: !this.options.name
```

Comment:
Also, I would reconsider even allowing empty strings, as name is usually required in licenses. Passing an empty string allows to skip it, so perhaps the parameter should be optional.